packages/application-shell/src/components/suspended-route/suspended-protected-route.tsx

```diff
@@ -0,0 +1,17 @@
+import { Suspense } from 'react';
+import LoadingSpinner from '@commercetools-uikit/loading-spinner';
+import { ProtectedRoute, type TProtectedRouteProps } from './protected-route';
+
+type TSuspendedProtectedRouteProps = TProtectedRouteProps;
+
+const SuspendedProtectedRoute: React.FC<TSuspendedProtectedRouteProps> = (
+  props
+) => {
+  return (
+    <Suspense fallback={<LoadingSpinner />}>
```

Comment:
In some cases a route opens a drawer. In that scenario the LoadingSpinner would increase CLS. 
For those few scenarios we could provide a way of overriding the default value of the fallback

```
type TSuspendedProtectedRouteProps = TProtectedRouteProps & {
  fallback: React.ReactNode
};

const SuspendedProtectedRoute: React.FC<TSuspendedProtectedRouteProps> = (
  { fallback = <LoadingSpinner /> }
) => {
  return (
    <Suspense fallback={fallback>
          <ProtectedRoute {...props} />
    </Suspense>
  );
};
```